### DISCUSSION TITLE
How introduce variance to series battery experiment

### DISCUSSION THREAD

**Discussion by skywo1f** [pybamm-team/PyBaMM#2565] [General]
I am trying to study how cell variance affects the battery pack as a whole. Is it possible to change the individual properties of cells when using the    
` "Number of cells connected in series to make a battery": 16.0,`
parameter?

Or would I change something in the experiment itself?

**Comment by valentinsulzer** [comment]
Not possible with PyBaMM, but you can use liionpack https://github.com/pybamm-team/liionpack

**Reply by skywo1f** [reply]
thanks! Ill check it out
